src/components/BreadList/BreadList.css

```diff
@@ -89,8 +89,9 @@
 @media screen and (min-width: 1025px){
   .BreadListWrapper h2 {
     margin-top: 2rem;
-    font-size: xx-large;
+    font-size: 2rem;
     margin-bottom: 1rem;
```

Comment:
This will be a merge conflict with the working code on my branch but since my styling has all migrated to the inline JSX with Tailwind, it should be easy to reconcile. If we need to change the font again, we can adjust as needed.